README.md

```diff
@@ -40,13 +40,13 @@ For all features and their usage see the [related section on the website](https:
 
 ## Related Projects
 
-Operator SDK plugin: https://github.com/operator-framework/java-operator-plugins
-
-Quarkus Extension: https://github.com/quarkiverse/quarkus-operator-sdk
-
-Spring Boot Starter: https://github.com/java-operator-sdk/operator-framework-spring-boot-starter
-
-Framework to implement Admission Controllers and Conversion Hooks: https://github.com/java-operator-sdk/kubernetes-webooks-framework
+* Quarkus Extension: https://github.com/quarkiverse/quarkus-operator-sdk
+* Spring Boot Starter: https://github.com/java-operator-sdk/operator-framework-spring-boot-starter
+* jenvtest: https://github.com/java-operator-sdk/jenvtest 
+  Support for integration testing again Kubernetes API Server
```

Comment:
```suggestion
  Lightweight Kubernetes API Server, optimized for integration testing in Java
```

Reply:
It's not really a lightweight API Server, it's THE Kubernetes Api server running

Reply:
Fair enough, but I don't think that the current description is very helpful. Why would I want to use this instead of running my tests against a real cluster for example? Your one-liner needs to capture that idea, somehow.

Reply:
changed.